ote/src/cljs/ote/ui/leaflet.cljs

```diff
@@ -88,4 +88,6 @@
 
 (defn background-tile-map []
   [TileLayer {:url "http://{s}.tile.osm.org/{z}/{x}/{y}.png"
-              :attribution "&copy; <a href=\"http://osm.org/copyright\">OpenStreetMap</a> contributors"}])
+              :attribution "&copy; <a href=\"https://opensource.org/licenses/MIT\">MIT</a>
+| &copy; <a href=\"https://www.maanmittauslaitos.fi/en/opendata-licence-cc40\">MML</a>
+| &copy; <a href=\"http://osm.org/copyright\">OpenStreetMap</a> contributors<br>"}])
```

Comment:
Should this same change be in ckan-service-viewer.cljs and place-search.cljs files?

Reply:
Updated in ckan-service-viewer. Removed dead code from place-search.cljs.